### DISCUSSION TITLE
Foundations for Young Adult Success: A Developmental Framework (2015)

### DISCUSSION THREAD

**Discussion by catherinet1** [life-itself/community#851] [Research]
Notes on: Nagaoka, Jenny, Camille A. Farrington, Stacy B. Ehrlich, and Ryan D. Heath with David W. Johnson, Sarah Dickson, Ashley Cureton Turner, Ashley Mayo, and Kathleen Hayes, *Foundations for Young Adult Success: A Developmental Framework* (2015), Chicago: University of Chicago CCSR.

![Screenshot 2023-10-06 at 10 39 20](https://github.com/life-itself/community/assets/97066487/4b98b904-59f6-4d86-bce9-b5b9e5429729)

# What is youth developmental 'success'?

- Success is defined as:
  - 'young people can fulfill individual goals and have the agency and competencies to influence the world around them' (p.1)
  - 'young people develop an awareness of themselves and of the wide range of options before them, competencies to pursue those options, and the ability to make good future choices for their lives as engaged citizens in the world' (pp.1-2)
- Key to note that socio-economic context plays a key role in youth development
- They also see youth development as integrating relation to self, relation to others, and relation to wider world (p.3)
- They identify three key factors of success (agency, competencies, and integrated identity) and four foundational components which pave the way towards developing those three factors of success (self-regulation, knowledge (and skills), mindsets, and values)
	- --> They say that 'self-regulation has numerous forms, including cognitive, emotional, behavioral, and attentional regulation'. Interestingly, nervous system or somatic self-regulation isn't named. But perhaps implicitly included within emotional self-regulation. 

# What is 'development'?

- 'Development is a natural, ongoing process that happens as young people observe the world, interact with others, and make meaning of their experiences. Regardless of the degree of adult guidance, children will still “develop” in some way, learning how to do things and coming to conclusions about themselves, their prospects, and their paths forward. They will develop some skills and preferences, and they will likely figure out what they need to know to get by' (p.4)

## What is 'deliberate development' (or 'intentional' development)?

The authors of this report don't use the term 'deliberately developmental' but they make an equivalent distinction between the kind of development that happens anyway regardless of any intervention or guidance (described above) and development that happens within the context of *intentionally* supportive structures and relationships. 

Two key elements:
- Developmental experiences
	- Provide opportunities for action and reflection
	- Need to provide age-appropriate conditions and stimuli, sensitive to the sequence of developmental progression in children, teens and young adults
	- Developmental potential is 'maximised' when these happen in context of supportive relationships and communities
- Developmental relationships

### How do humans learn?

Learning is _social_ and experiential, as well as neurological: 
- 'Learning is no longer understood as amassing facts in card-catalog fashion, or as trained behavioral responses to stimuli, but rather as changes in the complex neural interconnections in the brain; these neural connections are then “felt” as changes in our sense of understanding, or in our subjective experience of ourselves in the world.' (p.38)
- 'Critical to this process are strong, supportive, and sustained relationships with caring adults who can encourage young people to reflect on their experiences and help them to interpret those experiences in ways that expand their sense of themselves and their horizons. In this way, making sense of experience is an “unrelentingly social” process.' (p.44)

 The effectiveness of a community of practice:
 - 'As Halpern, Heckman, and Larson write in their report on adolescent learning, young people learn best when they are immersed in a “community of practice” that “allows its newest members to watch, listen to, and emulate…more experienced members. It provides models of action. It gives less experienced learners opportunity to see all the steps in addressing a problem before they are ready to accomplish these steps independently.”' (p.40)

We learn by reflecting on and making sense of our experiences:
- John Dewey: 'We do not learn from experience. We learn from reflecting on experience.' (p.44)
- 'Research has consistently shown that learning is accelerated and more readily transferred to other situations when people reflect on what happened, what worked, and what needs improving' (p.44)

## Developmental experiences

![Screenshot 2023-10-06 at 10 39 07](https://github.com/life-itself/community/assets/97066487/bc048843-7680-437d-9876-38c2ec481f0f)

#### Action

1) ENCOUNTER new places, new ideas, and models of what is possible (e.g. models of how to act, who they could become, communities they can aspire to join, inspiring/high quality work)
2) TINKER - play and explore and experience how the physical world works. Exercise creativity and imagination and thinking outside the box. 
3) PRACTISE - motivated and focused repetition with accurate feedback to build expertise over time
4) CHOOSE - grow towards responsible decision making: 'the ability to make constructive and respectful choices about personal behavior and social interactions based on consideration of ethical standards, safety concerns, social norms, the realistic evaluation of consequences of various actions, and the well-being of self and others' (CASEL definition)(p.43)
5) CONTRIBUTE in ways that are valuable to self and others as a way to feel a sense of meaning as well as exercise agency and develop self-confidence

#### Reflection and Meaning-making

1) DESCRIBING and EVALUATING
2) CONNECTING
3) ENVISIONING
4) INTEGRATING

**Comment by catherinet1** [comment]
Hi @matthewmccarthy11 and @BoazBF, I posted my notes on one of the articles (cf https://github.com/life-itself/labs/issues/173) here, so that you can start with this summary if you want, and so that we can share notes and comments here if we want.

welcoming discussion with others too

**Comment by catherinet1** [comment]
p.s. @asimong something you said in the call yesterday reminded me of this -- of learning as a process of both action and reflection. I think you said something very similar, that you see the need for both experience and then making sense/meaning of experience in order to learn or develop

**Reply by asimong** [reply]
I guess a huge amount has been written about this kind of thing. Kolb and [experiential learning](https://en.wikipedia.org/wiki/Experiential_learning) come to mind. There are myriad versions of this cycle: I see even Dragon Dreaming as closely related. [This presentation of mine from 2009](https://www.slideshare.net/asimong/eportfolio-values-ppt) has a related cycle (for the best view, download the ppt) The article on which this thread is based is yet another. It's just one among many — very many!

**Reply by catherinet1** [reply]
ah, I like your framework of competence = knowledge + capability + good choices/values; it's a helpful way to break it down and also include the ethical/values/agency dimension. thanks for sharing!

**Comment by catherinet1** [comment]
one question I have is, in understanding learning as a combination of action and reflection, **where does the kind of learning that leads to e.g. muscle memory fit in**? And, **are there ways in which the body learns that are different from this cycle of action and reflection**? 

For example: in my experience when I play a piece on the piano from memory, I don't think I ever 'reflected' on/made sense or meaning of my learning the piano piece, and sometimes when I think too much about what I'm playing I am no longer able to play it.

**Reply by asimong** [reply]
My sense here is that muscle memory is involved when a (physical) skill is being practised so often that it no longer needs conscious attention. Here I am typing away, not looking at my fingers, and the common words just appear without me thinking about them. That, to me, is one obvious example of muscle memory. It kicks in way later than the action-reflection cycles. Your piano example is also a very good one. Active sports involve such sensorimotor skills, too. To me they are just different kinds of learning, not really related.

**Comment by rufuspollock** [comment]
First, some simple appreciation on us doing this: @catherinet1 (and rest of team) just want to quickly say this is **awesome** 🎉 - just finding these frameworks and having a page for them is really useful and the additional notes is even better. 👏👏

**Comment by rufuspollock** [comment]
Re the framework: some v quick initial comments/thuoghts

- **deliberately developmental**: 👍 from notes it seems clearly "deliberately developmental"
- **focus on development (not results of that)**: this is a reflection from reading the notes and the overview of the framework: I'm wondering if having "outcome" goals is actually less useful than having more intermediate development goals. i.e. talking about agency (an outcome) is much vaguer than talking about self-regulation. Of course, we do "inner development" (or "cultivate being") because we think it has useful results ... however those results are varied, harder to measure etc. In addition, it can make development sound instrumental whereas i think we should be cautious about that. Think of an analogy like literacy (a classic cognitive developmental skill). With literacy i can do all kind of things: read books to wider my perspective on the world to learning programming. It seems to me focusing on simple literacy is better than focusing on the outcomes like "a broader perspective on the world".

**Reply by rufuspollock** [reply]
@matthewmccarthy11 note you can upvote posts which is more useful than emoji as it will move the posts up higher in the thread ... 😉

**Reply by matthewmccarthy11** [reply]
ah that makes sense
